microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md

```diff
@@ -0,0 +1,136 @@
+
+If you are on Windows, Linux, or macOS, consider using the Canary preview channel of Microsoft Edge as your default development browser.  The preview channels give you access to the latest features of Microsoft Edge DevTools.
+
+To report a problem with DevTools or ask for a new feature, file an issue at the new [Microsoft Edge DevTools feedback repo](https://github.com/MicrosoftEdge/DevTools).
+
+
+<!-- ====================================================================== -->
+## Join the Microsoft Edge DevTools community on GitHub
+
+<!-- Title: Head to the new DevTools repo on GitHub to send ideas, feedback, suggestions, and bugs -->
+<!-- Subtitle: You can file feedback, ask questions, and have discussions about DevTools in our GitHub repo. -->
+
+We're excited to announce the new [MicrosoftEdge/DevTools: Feedback and discussions about Microsoft Edge Developer Tools (github.com)](https://github.com/MicrosoftEdge/DevTools) repo on GitHub!  This new repo is a place for our community of developers to send us ideas, feedback, suggestions, and tips to improve DevTools. We're eager to discuss with you how DevTools can better serve your needs and meet the challenges you face every day in building great web experiences.
+
+![The DevTools feedback repo.](devtools-100-images/devtools-feedback-repo.png)
+
+Head to [our repo on GitHub](https://github.com/MicrosoftEdge/DevTools) to become a part of the Edge DevTools community.
+
+
+<!-- ====================================================================== -->
+## Filter heap snapshots summary by node type
+
+<!-- Title: Use new filters to focus on specific parts of a heap snapshot -->
+<!-- Subtitle: You can now filter by node type if, for example, you're only interested in the arrays or strings from the heap. -->
+
+When looking at all the objects in a heap snapshot in the **Memory** tool, it can be difficult to focus on specific objects or retaining paths.  Now in Microsoft Edge 100, you can use the new **Node Types** filter when looking at a heap snapshot, to focus on only specific types of nodes.  For example, to see only the arrays and string objects that are in the heap, select the **Array** and **String** entries in the **Node Types** filter.
+
+![Node Types in a heap snapshot in the Memory tool.](devtools-100-images/node-types-heap-snapshot.png)
+
+To learn more about taking a heap snapshot and analyzing the heap in DevTools, see [Record heap snapshots using the Memory tool](../../../memory-problems/heap-snapshots.md).
```

Comment:
Thank you for adding a new section to the linked doc. The additional section looks great to me.